python/ray/air/tests/test_api.py

```diff
@@ -72,8 +71,11 @@ def test_scaling_config():
     with pytest.raises(ValueError):
         DummyTrainer(scaling_config=True)
 
+    with pytest.raises(ValueError):
+        DummyTrainer(scaling_config={})
+
```

Comment:
nice

Reply:
Actually, should we keep dict support for backwards compatibility (as Ray AIR was included in the 1.13 release) or alternatively raise an actionable `DeprecationWarning` instead?